apps/clock_info/lib.js

```diff
@@ -298,7 +298,7 @@ exports.addInteractive = function(menu, options) {
   };
   const focus = () => {
     let redraw = true;
-    Bangle.CLKINFO_FOCUS=true;
+    Bangle.CLKINFO_FOCUS = (Bangle.CLKINFO_FOCUS || 0) + 1;
```

Comment:
Please could we make this just `(0|Bangle.CLKINFO_FOCUS) + 1;`?

It's just a bit more robust (eg if `Bangle.CLKINFO_FOCUS=="1"` the result of running this code is the string `"11"`)

Reply:
Sure, sorted 👍